client/my-sites/theme/theme-features-card.js

```diff
@@ -27,7 +32,15 @@ const ThemeFeaturesCard = ( { isWpcomTheme, siteSlug, features, translate, onCli
 							{ ! isWpcomTheme ? (
 								<span>{ name }</span>
 							) : (
-								<a href={ `/themes/filter/${ term }/${ siteSlug || '' }` }>{ name }</a>
+								<a
+									href={ localizeUrl(
+										`https://wordpress.com/themes/filter/${ term }/${ siteSlug || '' }`,
```

Comment:
With the switch to full URL, I'm unable to test the client-side navigation as it's now an external link for the dev and calypso.live environments (due to running from a different host).

I came across the following helper for the themes paths - https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/themes/helpers.js#L61. It might be worth trying if it can be used in this case.